src/main/plugins/org.dita.base/xsl/common/functions.xsl

```diff
@@ -60,6 +60,13 @@ See the accompanying LICENSE file for applicable license.
            else $doc
     "/>
   </xsl:function>
+  
+  <!-- $href (might be @href or @copy-to) has absolute or external value that should not be changed -->
+  <xsl:function name="dita-ot:unmodifiable-uri" as="xs:boolean">
```

Comment:
The name could instead be `dita-ot:is-external`, because that's what we're testing for.